src/design/merge-css.md

```diff
@@ -2,21 +2,21 @@
 title: Merging CSS Files
 ---
 
-As part of an effort to optimize your site and reduce page load time, you can reduce the number of separate CSS files by merging them into a single condensed file. If you open a merged CSS file, you’ll find one continuous stream of text, with line breaks removed. Because you can’t edit the merged file, it’s best to wait until you are out of the development mode, and no longer making frequent changes to the CSS.
+As part of an effort to optimize your site and reduce page load time, you can reduce the number of separate CSS files by merging them into a single, condensed file. If you open a merged CSS file, you will find one continuous stream of text, with line breaks removed. You cannot edit the merged file, so it is best to wait until you are out of the development mode and no longer making frequent changes to the CSS.
 
 {:.bs-callout-info}
 CSS files can be merged only when working in [Developer Mode]({% link magento/installation-modes.md %}).
 
-## To merge CSS files:
+## Merge your CSS files
 
 1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Advanced**, choose **Developer**.
+1. In the left panel, expand **Advanced** and choose **Developer**.
```

Comment:
```suggestion
1. In the left panel, expand ![]({% link images/images/btn-expand.png %}) **Advanced** and choose **Developer**.
```

Reply:
this is the panel and doesn't use the same icon as the page.